### PR TITLE
SCAL DQM monitoring in CMSSW76X

### PR DESCRIPTION
**Pull request by sachikot**
updated scal online dqm monitoring to cmssw76X

### CODE REVIEW

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
+1

File: DQMServices/Components/src/DQMScalInfo.cc [hunk 1]
```diff
@@ -25,6 +31,7 @@ DQMScalInfo::DQMScalInfo(const edm::ParameterSet& ps)
   gtCollection_        = consumes<L1GlobalTriggerReadoutRecord>(parameters_.getUntrackedParameter<std::string>("gtCollection","gtDigis"));
   dcsStatusCollection_ = consumes<DcsStatusCollection>(parameters_.getUntrackedParameter<std::string>("dcsStatusCollection","scalersRawToDigi"));
   l1tscollectionToken_ = consumes<Level1TriggerScalersCollection>(parameters_.getUntrackedParameter<std::string>("l1TSCollection", "scalersRawToDigi"));
+  lumicollectionToken_ = consumes<LumiScalersCollection>(parameters_.getUntrackedParameter<std::string>("lumiCollection", "scalersRawToDigi"));
```
Comment (davidlange6) [inline DQMServices/Components/src/DQMScalInfo.cc]: this should be an InputTag not a string. (I guess that is true for all of them, but we should not make the problem bigger)

**Comment by davidlange6** [comment]
ping @sachikot

**Comment by deguio** [comment]
ciao @sachikot 
can you please fix the code as @davidlange6 suggested?
otherwise this PR won't be merged.
thanks,
F.

**Comment by sachikot** [comment]
Dear Federico,

Sorry for late response. I am trying to fix the code, could you please wait a little bit?

Best regards,
Sachiko

________________________________

**Comment by danduggan** [comment]
Hi Sachiko,

I'm very sorry for the delay in getting the code to you. In the end the fix
is rather straightforward:
  gtCollection_        =
consumes<L1GlobalTriggerReadoutRecord>(parameters_.getUntrackedParameter<edm::InputTag>(std::string("gtCollection"),std::string("gtDigis")));
  dcsStatusCollection_ =
consumes<DcsStatusCollection>(parameters_.getUntrackedParameter<edm::InputTag>(std::string("dcsStatusCollection"),std::string("scalersRawToDigi")));
  l1tscollectionToken_ =
consumes<Level1TriggerScalersCollection>(parameters_.getUntrackedParameter<edm::InputTag>(std::string("l1TSCollection"),
std::string("scalersRawToDigi")));

Can you make this update in your code, quickly test and then update the
branch?

Thanks,
Dan

**Comment by deguio** [comment]
please rebase.
thanks,
F.

**Comment by deguio** [comment]
to be rebased

**Comment by deguio** [comment]
the changes should rather be:

    l1tscollectionToken_ = consumes<Level1TriggerScalersCollection>(parameters_.getUntrackedParameter<edm::InputTag>("l1TSCollection", edm::InputTag("scalersRawToDigi")));

**Pull request closed without merging**